vendor-overwrites/codemirror/addon/lint/stylelint-config.js

```diff
@@ -0,0 +1,165 @@
+window.stylelintDefaultConfig = (defaultSeverity => ({
```

Comment:
this file doesn't belong to codemirror so it shouldn't be in vendor-overwrites, move it edit/ or js/ depending on whether it's used somewhere besides the editor